src/delaunay.js

```diff
@@ -95,10 +93,13 @@ export default class Delaunay {
   }
   renderHull(context) {
     const buffer = context == null ? context = new Path : undefined;
-    const {hull} = this;
-    let node = hull;
-    context.moveTo(node.x, node.y);
-    while (node = node.next, node !== hull) context.lineTo(node.x, node.y);
+    const hull = this.hull, points = this.points;
+    let h = hull[hull.length - 1];
+    context.moveTo(points[2 * h], points[2 * h + 1]);
```

Comment:
Nit: the `moveTo` seems to be redundant if we have `closePath` below.
Nit 2: code style, prefer destructuring: `const {hull, points} = this;`

Reply:
correct!